js/everything.js

```diff
@@ -357,6 +364,11 @@ function getCurrentVideoID() {
     return v;
 }
 
+function getCurrentTimePosition() {
+    var t = getParameterByName(window.location.search, "t");
```

Comment:
We should be trying to `parseInt(t, 10)` here, and only return the value if it's valid (ie: not `NaN`)